### PR TITLE
[util] Adapt flash size reduce/check scripts for changes in flash params

### PR DESCRIPTION
**Pull request by vogelpi**
We've recently changed the flash parametrization in lowRISC/OpenTitan#4534. This PR adapts the flash size reduce/check scripts required for smaller FPGAs accordingly.

This fixes lowRISC/OpenTitan#4638.

### CODE REVIEW

**Comment by tjaychen** [comment]
ah thanks @vogelpi 
do you think we should run this script as part of CI just to double check for errors in the future?

**Comment by vogelpi** [comment]
I would say we should get rid of it as soon as #4329 is merged. And over there we definitely need CI checks.

**Comment by vogelpi** [comment]
Hey @tjaychen , would you mind to approve this PR please? I think even if we decide to remove script soon, it makes sense to merge this PR. Thanks!

**Review by imphil — APPROVED** [review]
Let's get it in for now -- better to have a working script than a broken one. And we can then discuss the path forward with this and similar approaches with less time pressure.

**Comment by vogelpi** [comment]
Thanks @imphil :+1:

**Comment by tjaychen** [comment]
sorry! i didn't realize i forgot to approve it!
thanks @imphil and thanks @vogelpi

**Pull request merged**
